site/docs/main/restic.md

```diff
@@ -48,17 +51,19 @@ hostPath:
   path: /opt/rke/var/lib/kubelet/pods
 ```
 
+Now you can [install Velero](#install-restic) with the restic integration.
+
 **OpenShift**
 
-The restic containers should be running in a `privileged` mode to be able to mount the correct hostpath to pods volumes.
+To be able to mount the correct hostpath to pods volumes, run the restic containers in a `privileged` mode.
```

Comment:
suggest:
```
To mount the correct hostpath to pod volumes, run the restic containers in `privileged` mode.
```

Reply:
would it be more accurate to say `restic pods` instead of `restic containers`? or do you mean the containers, and not a pod in this case?

Reply:
@ashish-amarnath see above q :)

Reply:
sorry for the latency :)
Yes, restic pods would be accurate.

Reply:
thanks, i updated this to say pods to be consistent.